### PR TITLE
chapter9

### PR DESCRIPTION
**Pull request by ljj7975**

### CODE REVIEW

File: Chapter_9/sagemaker/README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+## Inferencing using SageMaker
+
+In the second half of the chapter, we introduce SageMaker-based deployment. We discuss how to create inference endpoints for TF, PyTorch, and ONNX models. Additionally, the endpoints will be optimized using Amazon SageMaker Neo and El accelerators. The next topic is on how to set up automatic scaling for the inference endpoints running on SageMaker. Finally, we will wrap up the chapter by describing how to host multiple models in a single SageMaker inference endpoint.
+
+* [Inference endpoint for TensorFlow](./tf-inference.ipynb)
+* [Inference endpoint for PyTorch](./pytorch-inference.ipynb)
+* [Elastic Inference (EI) accelerator](./ei-inference.ipynb)
+
+# TODO: what is the diff between ei.ipynb and ei-inference.ipynb. Please update the link accordingly. Also, the notebooks are not polished (contains bunch of commented lines, no explanations and errors). Please take a look and make sure they are cleaned up
```
Comment (ljj7975) [inline Chapter_9/sagemaker/README.md]: @leninworld

File: Chapter_9/eks/README.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+## Inferencing using Elastic Kubernetes Service
+
+This first section of this chapter describes EKS-based approach. We introduce how to create inference endpoints for TensorFlow (TF) and PyTorch models and deploy them using EKS. We will also introduce Elastic Inference (EI) accelerator which can increase the throughput while reducing the cost. EKS clusters have pods that host the inference endpoints as webservers. As the last topic for EKS-based deployment, we will introduce how the pods can be scaled horizontally for the dynamic incoming traffic.
+
+# TODO: please add notebooks
```
Comment (ljj7975) [inline Chapter_9/eks/README.md]: @leninworld

File: Chapter_9/sagemaker/README.md (hunk 1 above)
Comment (leninworld) [reply Chapter_9/sagemaker/README.md]: renamed the file (the other file was to directly deploy a TF model file and create endpoint.

File: Chapter_9/eks/README.md (hunk 2 above)
Comment (leninworld) [reply Chapter_9/eks/README.md]: @ljj7975  We are not adding any EKS scripts. We only upload the SageMaker. So EKS folder is to be removed.

Comment (leninworld) [reply Chapter_9/eks/README.md]: @ljj7975 EKS related details (links) added

**Pull request merged**